cmd/atlas/internal/mux/mux.go

```diff
@@ -0,0 +1,72 @@
+package mux
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"ariga.io/atlas/sql/schema"
+)
+
+type (
+	// Mux is used for routing dsn to correct provider.
+	Mux struct {
+		providers map[string]func(string) (*Driver, error)
+	}
+
+	// Driver implements the Atlas interface.
+	Driver struct {
+		*sql.DB
+		Differ    schema.Differ
+		Execer    schema.Execer
+		Inspector schema.Inspector
+	}
+)
+
+// Close release all resources taken by Driver.
+func (a *Driver) Close() error {
+	return a.DB.Close()
+}
```

Comment:
not needed.

Reply:
👍 
why? cli closing is good enough?

Reply:
or just using the API directly?

Reply:
when you embed a type you can call its methods directly